src/Ushahidi/Modules/V5/Requests/CSVRequest.php

```diff
@@ -26,10 +26,14 @@ public function rules(Request $request)
             return [];
         }
     }
+    
     public function getMaxBytes()
     {
         // To Do: make it config
-        return 20000;
+        // David Losada update 2024-03-15: originally 20KB, that size is arbitrary and not useful to anyone.
+        //    Setting to 1TiB, instead
+        //    Let's make it something reasonable when the work to make this a config value happens.
+        return 1024 * 1024 * 1024 * 1024;
```

Comment:
I think this is a bit high. Can I suggest somewhere in the range of 20mb - 50mb?